### PR TITLE
Fix conformance tests for non aws and packet providers

### PR DESCRIPTION
**Pull request by nikhita**
Fix conformance tests for non aws and packet providers.

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/assign @alvaroaleman

### CODE REVIEW

File: api/hack/ci/ci-run-minimal-conformance-tests.sh [hunk 1]
```diff
@@ -212,10 +212,10 @@ fi
 
 echodate "Starting conformance tests"
 
-if [[ $provider = "aws" ]]; then
-  EXTRA_ARGS="-aws-access-key-id=${AWS_E2E_TESTS_KEY_ID}
-     -aws-secret-access-key=${AWS_E2E_TESTS_SECRET}"
-elif [[ $provider = "packet" ]]; then
```
Comment (alvaroaleman) [inline api/hack/ci/ci-run-minimal-conformance-tests.sh]: This does look correct to me, how is it not working?

Comment (nikhita) [reply api/hack/ci/ci-run-minimal-conformance-tests.sh]: because `EXTRA_ARGS` becomes an unbound variable if the provider is not aws or packet. :(

Comment (nikhita) [reply api/hack/ci/ci-run-minimal-conformance-tests.sh]: Or wait! We do check for the provider before this. Checking if the logs with this error on slack were stale ones.

Comment (nikhita) [reply api/hack/ci/ci-run-minimal-conformance-tests.sh]: Updated.

**Pull request merged**